src/transport/SecureSessionTable.cpp

```diff
@@ -183,6 +220,100 @@ SecureSession * SecureSessionTable::EvictAndAllocate(uint16_t localSessionId, Se
 
 void SecureSessionTable::DefaultEvictionPolicy(EvictionPolicyContext & evictionContext)
 {
+    evictionContext.Sort([&evictionContext](const SortableSession & a, const SortableSession & b) {
+        //
+        // Sorting on Key1
```

Comment:
expand comment as after we merge the PR, "Key1" is not as well defined. 
The text of "Sessions on fabrics with more sessoions are more likely to get evicted" seems useful instead of Key1.

Reply:
Definitions of what each keyN does is already specified in the description of this function, so duplicating it here would seem redundant.